client/src/lib/styled/interface.tsx

```diff
@@ -0,0 +1,27 @@
+import { nightable } from './colors';
+
+interface IAppTheme {
+  pBackground: string,
+  pText: string,
+}
+
+const night: IAppTheme = {
+  pBackground: nightable.background.night,
+  pText: nightable.text.night,
+}
+
+const day: IAppTheme = {
+  pBackground: nightable.background.day,
+  pText: nightable.text.day,
+}
+
+const party: IAppTheme = {
```

Comment:
should probably use `nightable` (themeable?)

Reply:
Pas vraiment, party cest un theme in itself, qui a pas de verion night.

A moins qu' on aie partyDay et partyNight ?